pynitrokey/cli/fido2.py

```diff
@@ -233,15 +233,20 @@ def list_credentials(serial, pin):
     # Get amount of registered creds from first key in list (Same trick is used in the CredentialManager)
     local_print(f"There are {cred_count} registered credentials")
 
-    for reliable_party in reliable_party_list:
-        reliable_party_hash = reliable_party.get(CredentialManagement.RESULT.RP_ID_HASH)
+    for reliable_party_result in reliable_party_list:
+        reliable_party = reliable_party_result.get(CredentialManagement.RESULT.RP)
+        reliable_party_hash = reliable_party_result.get(CredentialManagement.RESULT.RP_ID_HASH)
+        local_print("-----------------------------------")
+        local_print(f"{reliable_party['name']}: ")
         for cred in cred_manager.enumerate_creds(reliable_party_hash):
-            local_print("-----------------------------------")
-            cred_id = cred.get(CredentialManagement.RESULT.CREDENTIAL_ID)
+            cred_id = cred.get(CredentialManagement.RESULT.CREDENTIAL_ID)['id']
             cred_user = cred.get(CredentialManagement.RESULT.USER)
-            local_print(f"ID: {cred_id['id'].hex()}, user: {cred_user}")
+            if cred_user['name'] == cred_user['displayName']:
+                local_print(f"- {cred_id.hex()}: {cred_user['name']}\n")
+            else:
+                local_print(f"- {cred_id}: {cred_user['displayName']} ({cred_user['name']})\n")
```

Comment:
Do we need the trailing `\n`?

Reply:
I'd recommend leaving them there since they make it a lot easier to distinguish keys